src/components/Dashboard.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC, useState } from 'react'
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import { Typography } from '@mui/material'
+
+
+const Dashboard: FC = () => {
+
+  //state to store users' input
+  const [formData, setFormData] = useState('')
+
+  //save current input in state, before user submits
+  const onChange = (e: any )=> {
+    setFormData(e.target.value)
```

Comment:
If you plan on only setting the formData with this function, it may be best to have this within the component as the type will be implied. Functionality is the same but may be cleaner and more semantic!

Reply:
Could you show me how you've done this at the start of standups?